velodyne_driver/include/velodyne_driver/driver.h

```diff
@@ -78,6 +79,8 @@ class VelodyneDriver
     double time_offset;              // time in seconds added to each velodyne time stamp
     bool enabled;                    // polling is enabled
     bool timestamp_first_packet;
+    int32_t ok_scan_age_ns;
+    int32_t warn_scan_age_ns;
```

Comment:
fwiw, since the distinction between ok, warn & error is arbitrary, we might get away with just reporting warning or ok, and reduce to just one extra var / setting